package.json

```diff
@@ -13,10 +13,11 @@
     "example": "examples"
   },
   "scripts": {
-    "test:firefox": "scripts/test --browsers Firefox --single-run",
-    "test:chrome": "scripts/test --browsers Chrome --single-run",
```

Comment:
+1 we should move this to a separate PR, so we can run something like `npm run test:firefox -- --single-run`. So, maybe, we can include `travis`.(?)